content/en/docs/ops/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -83,12 +85,12 @@ $ helm upgrade ztunnel istio/ztunnel -n istio-system
 
 ### Upgrade the CNI Component
 
-The Istio CNI agent is responsible for detecting the pods that belong to the ambient mesh, and configuring the traffic redirection between pods and the ztunnel DaemonSet. It is not part of the data plane or control plane.
+The Istio CNI agent is responsible for detecting the pods that belong in the ambient mesh, informing ztunnel that proxy ports should be established within added pods, and configuring traffic redirection within the pod network namespace. It is not part of the data plane or control plane.
```

Comment:
```suggestion
The Istio CNI agent is responsible for detecting the pods are added to the ambient mesh, informing ztunnel that proxy ports should be established within added pods, and configuring traffic redirection within the pod network namespace. It is not part of the data plane or control plane.
```

Reply:
use add to ambient mesh for consistency.

Also, i am not sure users need to know proxy ports other than in a troubleshooting docs and detailed "how it works" docs.